src/http_rest_api_handler.cpp

```diff
@@ -424,6 +383,56 @@ static Status validateContentFieldsEmptiness(KFSTensorInputProto* input) {
     return StatusCode::OK;
 }
 
+static bool isInputEmpty(::KFSRequest::InferInputTensor* input) {
```

Comment:
This function acts in a way that input can never be nullptr (there is no check if input is nullptr before proceeding).
```suggestion
static bool isInputEmpty(const ::KFSRequest::InferInputTensor& input) {
```
Therefore I'd suggest to change pointer to reference. 
I'd also add `const` since you never change the underlying data.